tests/tpch/dask_queries.py

```diff
@@ -66,7 +66,7 @@ def query_2(dataset_path, fs, scale):
     )
     part_filtered = part_filtered[
         (part_filtered["p_size"] == var1)
-        & (part_filtered["p_type"].astype(str).str.endswith(var2))
+        & (part_filtered["p_type"].astype("string[pyarrow]").str.endswith(var2))
```

Comment:
why is this cast necessary in the first place?

![image](https://github.com/coiled/benchmarks/assets/8629629/aafdbc48-8015-42cf-8028-c529cdceb591)

```suggestion
        & (part_filtered["p_type"].str.endswith(var2))
```

Reply:
Very good point, I was just passing by here. The cast doesn't seem necessary